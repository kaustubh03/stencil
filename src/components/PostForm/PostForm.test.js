import React from "react";
import renderer from "react-test-renderer";
import Postform from ".";
import configureStore from "../../redux/configureStore";
const store = configureStore();

it("Postform: default", () => {
  const component = renderer.create(<Postform store={store} />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
