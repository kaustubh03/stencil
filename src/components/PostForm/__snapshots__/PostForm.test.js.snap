// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Postform: default 1`] = `
<div
  className="post_container"
>
  <h2
    className="post_heading"
  >
    Create Post
  </h2>
  <form
    className="form"
    onSubmit={[Function]}
  >
    <input
      placeholder="Enter Post Title"
      required={true}
      type="text"
    />
    <br />
    <br />
    <textarea
      cols="28"
      placeholder="Enter Post"
      required={true}
      rows="5"
    />
    <br />
    <br />
    <button>
      Post
    </button>
  </form>
</div>
`;
